sympy/core/relational.py

```diff
@@ -1075,8 +1087,48 @@ def _n2(a, b):
             return dif
 
 
+@dispatch(Number, Pow)
+def _eval_is_ge(lhs, rhs): # noqa:F811
+    if lhs:
+        base, exp = rhs.as_base_exp()
+        if exp.is_nonpositive and is_ge(base, 1/lhs):
+            return True
```

Comment:
We should try to avoid creating new Basic instances like `1/lhs`.

Is the logic here correct?

If we have
```
1/2 >= 5 ** 0
```
Then
```
lhs = 1/2
base = 5
exp = 0
```
So `exp.is_nonpositive = True` and `base >= 1/lhs` so we return True but the inequality is not true.

Reply:
I think that is why the tests are failing...I have to look more closely at this. 

As to creating different objects, that is the purpose of working with only Number but perhaps I should have limited that to Rational and then we wouldn't be making a new type.